apps/aecore/test/aec_chain_sim.erl

```diff
@@ -633,10 +695,17 @@ trees(Blocks) ->
     end.
 
 update_trees(Txs, Trees, Height) ->
-    Env = aetx_env:tx_env(Height),
-    {ok, _, [], NewTrees, Evs} = aec_trees:apply_txs_on_state_trees_strict(Txs, Trees, Env),
+    Env0 = aetx_env:tx_env(Height),
+    Env1 = aetx_env:set_dry_run(Env0, true),
```

Comment:
I see this is the change that exposes error messages. It is a bit hacky and worth a line of comment?